elements/pfe-modal/test/pfe-modal_test.html

```diff
@@ -51,6 +59,14 @@ <h2 slot="pfe-modal--header">Modal with a header</h2>
           assert.isTrue(modalWindow.hasAttribute('hidden'));
         
         });
+        
+        test('it should remove the hidden attribute from the host on open', done => {
+          // get the modal and fire the open function
```

Comment:
@LyndseyR You'll need to replace the comments I put there with the code to do those things

Reply:
@castastrophe I added a test that when the trigger was clicked the modal window appeared and the pfe-modal hidden attribute wasn't appearing. Is this enough?

Reply:
I love it!  That's perfect.  I also really like that you included the shadow DOM test on the pfe-modal__window. Thank you so much!